util/windows/windows_api.go

```diff
@@ -34,24 +36,34 @@ type MEMORYSTATUSEX struct {
 	AvailExtendedVirtual uint64
 }
 
+// PDH_FMT_COUNTERVALUE_DOUBLE XXX
 type PDH_FMT_COUNTERVALUE_DOUBLE struct {
 	CStatus     uint32
 	DoubleValue float64
 }
 
+// PDH_FMT_COUNTERVALUE_ITEM_DOUBLE XXX
 type PDH_FMT_COUNTERVALUE_ITEM_DOUBLE struct {
 	Name     *uint16
 	FmtValue PDH_FMT_COUNTERVALUE_DOUBLE
 }
 
 const (
+	// ERROR_SUCCESS XXX
```

Comment:
You can just add comments to enclosing `const` block to suppress lint, not each one.